pxtsim/logger.ts

```diff
@@ -0,0 +1,67 @@
+namespace pxt {
```

Comment:
I take it there's no nice, shared location both pxtsim and pxtlib can consume this from as a single file?

Reply:
nope...